### ISSUE TITLE
LF Project labels

### ISSUE THREAD

**Issue by dankohn** [cncf/landscapeapp#248]
When we switched to the member landscape, we lost the functionality where organization entries associated with the Linux Foundation would show as `LF Project`. We used to track here: https://github.com/cncf/landscape/blob/4a3db6024cae9d85881a71bc445391a7827becd3/members.yml#L381

Could we please add it back, preferably in an automated fashion?

All of those organizations such as https://www.crunchbase.com/organization/continuous-delivery-foundation-cdf are sub-foundations of https://www.crunchbase.com/organization/the-linux-foundation. In settings.yml, you already have:
```yml
  linux_foundation:
    name: LF
    label: Project
    funding: Linux Foundation
```
Please add functionality to add the `LF Project` label to LF sub-organizations automatically.l The exception should be CNCF projects, which should be labeled as `CNCF Incubating` (instead of `Cloud Native Incubating` that we have now).

**Comment by jordinl83** [comment]
@dankohn How is this `LF Project` label supposed to be used in the UI?

**Comment by dankohn** [comment]
Look at: https://landscape.cncf.io/category=cloud-native-network&format=card-mode&grouping=category&organization=fd-io,cloud-native-computing-foundation-cncf

CNI should say "CNCF Incubating" with CNCF in bold. NSM should say "CNCF Sandbox" with CNCF in bold. FD.io should say "LF Project" with LF in bold.

**Comment by jordinl83** [comment]
@dankohn Regarding the labels that should be "CNCF" instead of "Cloud Native", these are set in https://github.com/cncf/landscape/blob/master/settings.yml#L35-L57, the `prefix` from the settings should be set to "CNCF".

**Comment by dankohn** [comment]
Thanks for letting me know.

**Comment by jordinl83** [comment]
@dankohn Other than adding the `LF Project` label to the modal, should the label be a link? Also, should there be a filter on the left side for `Linux Foundation` projects?

**Comment by dankohn** [comment]
Yes, let's add a new, undocumented mode similar to https://landscape.cncf.io/enduser=yes. Clicking on the badge should add `https://landscape.cncf.io/format=card-mode&grouping=organization&linuxfoundation=yes` where it shows each project that is under the Linux Foundation, grouped by organization. Note that CNCF projects should also be shown.
